tests/Testcontainers.Qdrant.Tests/X509CertificateGenerator.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace Testcontainers.Qdrant;
+
+public static class X509CertificateGenerator
+{
+    public record PemCertificate(string Certificate, string PrivateKey);
+    
+    public static PemCertificate Generate(string subjectName)
```

Comment:
Cannot we use .NET's built-in capabilities, like:

```csharp
var request = new CertificateRequest("CN=MyCertificate", RSA.Create(2048), HashAlgorithmName.SHA256, RSASignaturePadding.Pkcs1);
request.CertificateExtensions.Add(new X509BasicConstraintsExtension(false, false, 0, true));
var certificate = request.CreateSelfSigned(DateTimeOffset.UtcNow, DateTimeOffset.UtcNow.AddYears(1));
return new PemCertificate(certificate.ExportCertificatePem(), certificate.GetRSAPrivateKey().ExportPkcs8PrivateKeyPem());
```

Reply:
That's much simpler, will update 👍